### PR TITLE
ntr skeletal fluorosis issue #772

### PR DESCRIPTION
**Pull request by laurenechan**
Addition of skeletal fluorosis, more information on term in issue #772

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
Lauren added an ID range in the owl file in this PR: https://github.com/monarch-initiative/mondo/pull/778

**Comment by nicolevasilevsky** [comment]
@cmungall do you think the problem is with her protege settings?

**Comment by laurenechan** [comment]
@cmungall Sorry about that, I updated the ID# to reflect my range. Let me know if there are any other issues!

**Comment by nicolevasilevsky** [comment]
@cmungall I resolved the conflicts, this should be ready to merge

**Comment by cmungall** [comment]
conflicts.

Remember to do diffs before commiting, owl-axioms line should not change

**Comment by nicolevasilevsky** [comment]
I'll resolve the conflicts with @laurenechan, it will give an opportunity to teach her how to resolve conflicts

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -428276,6 +428276,14 @@ xref: http://orcid.org/0000-0001-9969-8610
 is_a: MONDO:0005500 ! congenital disorder of glycosylation type I
 created_by: http://orcid.org/0000-0003-2338-2550
 
+[Term]
+id: MONDO:0400003
+name: skeletal fluorosis
+def: "A condition that results from excessive fluoride ingestion leading to fluoride accumulation in the bone progressively over many years. The early symptoms of skeletal fluorosis, include stiffness and pain in the joints. In severe cases, the bone structure may change and ligaments may calcify, with resulting impairment of muscles and pain." [PMID:3295994]
+synonym: "fluorosis of the skeleton" EXACT []
+is_a: MONDO:0005066 ! metabolic disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: would be good to have axioms link to uberon:bone, but can be done later...

**Pull request merged**